internal/rbd/rbd_journal.go

```diff
@@ -585,7 +588,7 @@ func RegenerateJournal(
 		return "", err
 	}
 
-	rbdVol.RequestName = requestName
+	rbdVol.RequestName = volumeAttributes[pvNameKey]
```

Comment:
pv name will be removed from the volumeAttributers in sometime as i mentioned ealier